mono/mini/unwind.c

```diff
@@ -518,6 +518,8 @@ mono_unwind_frame (guint8 *unwind_info, guint32 unwind_info_len,
 				   mgreg_t **save_locations, int save_locations_len,
 				   guint8 **out_cfa)
 {
+	MONO_SCOPE_ENABLE_JIT_WRITE();
```

Comment:
Again, odd place to need this? In stack unwinding logic?

Reply:
Not sure off the top of my head. Looking into it.

Reply:
Looks like this one isn't needed. All of our runtime tests run just fine without this. I'll remove it.